### ISSUE TITLE
Project management

### ISSUE THREAD

**Issue by writemonkey** [writemonkey/wm3#25]

**Comment by Negreuj** [comment]
Yes, some kind of project management that would allow you to connect your writing projects to a time line and keep them in a prioritized order of completion would be an enormously beneficial feature!
